content/en/agent/autodiscovery/integrations.md

```diff
@@ -0,0 +1,541 @@
+
+{{% /tab %}}
+{{% tab "Docker Label" %}}
+
+You can store your Integrations templates as docker labels since the Agent detects if it's running on Docker and automatically searches all labels for Integration templates. Autodiscovery expects labels to look like these examples, depending on the file type:
+
+**Dockerfile**:
+
+```yaml
+LABEL "com.datadoghq.ad.check_names"='[<CHECK_NAME>]'
+LABEL "com.datadoghq.ad.init_configs"='[<INIT_CONFIG>]'
+LABEL "com.datadoghq.ad.instances"='[<INSTANCE_CONFIG>]'
+LABEL "com.datadoghq.ad.logs"='[<LOGS_CONFIG>]'
+```
+
+**docker-compose.yaml**:
+
+```yaml
+labels:
+  com.datadoghq.ad.check_names: '[<CHECK_NAME>]'
+  com.datadoghq.ad.init_configs: '[<INIT_CONFIG>]'
+  com.datadoghq.ad.instances: '[<INSTANCE_CONFIG>]'
+  com.datadoghq.ad.logs: '[<LOGS_CONFIG>]'
+```
+
+**docker run command**:
+
+```shell
+-l com.datadoghq.ad.check_names='[<CHECK_NAME>]' -l com.datadoghq.ad.init_configs='[<INIT_CONFIG>]' -l com.datadoghq.ad.instances='[<INSTANCE_CONFIG>]' -l com.datadoghq.ad.logs='[<LOGS_CONFIG>]'
+```
+
+**Docker Swarm**:
+
+When using Swarm mode for Docker Cloud, labels must be applied to the image:
+
+```yaml
+version: "1.0"
+services:
+...
+  project:
+    image: '<IMAGE_NAME>'
+    labels:
+      com.datadoghq.ad.check_names: '[<CHECK_NAME>]'
+      com.datadoghq.ad.init_configs: '[<INIT_CONFIG>]'
+      com.datadoghq.ad.instances: '[<INSTANCE_CONFIG>]'
+      com.datadoghq.ad.logs: '[<LOGS_CONFIG>]'
+
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Find below two sections giving examples for all of the previous method:
+
+* [Example Datadog-Redis Integration](#example-datadog-redis-integration)
+* [Example Datadog-Apache and Datadog-HTTP checks Integrations](#example-datadog-apache-and-http-checks-integrations)
+
+## Example Datadog-Redis Integration
+
+{{< tabs >}}
+{{% tab "File" %}}
+
+Redis is one of the default Autodiscovery template packaged with the Agent, so you don't need to mount this file. Here's the auto-conf `redis.yaml` template packaged with the Agent:
+
+```yaml
+ad_identifiers:
+  - redis
+
+init_config:
+
+instances:
+
+  - host: "%%host%%"
+    port: "6379"
+```
+
+It looks like a minimal [Redis integration configuration][1], but notice the `ad_identifiers` option. This required option lets you provide container identifiers. Autodiscovery applies this template to any containers on the same host that run an `redis` image.
+
+**See the dedicated [Autodiscovery Identifier][2] documentation to learn more.
+
+Now let's suppose that your Redis requires an additional `password` when accessing his stats endpoint and you want to correctly flag logs coming out of it. In order to take into account this new logic:
+
+1. Create a `autodiscovery.d/` folder on your host.
+2. Add the following custom auto-configuration named `redisdb.yaml`file to this folder:
+  ```yaml
```

Comment:
Formatting issue, ```yaml no taken in account in the preview.

Reply:
it is, it's just our yaml highlighting is quite light.